storage/service_account/hmac/hmac_test.go

```diff
@@ -119,21 +119,22 @@ func TestDeactivateKey(t *testing.T) {
 
 func TestGetKey(t *testing.T) {
 	tc := testutil.SystemTest(t)
-	key, err := createTestKey(tc.ProjectID)
-	defer deleteTestKey(key)
-	if err != nil {
-		t.Errorf("Error in key creation: %s", err)
-	}
-
 	testutil.Retry(t, 10, 10*time.Second, func(r *testutil.R) {
+		key, err := createTestKey(tc.ProjectID)
+		defer deleteTestKey(key)
+		if err != nil {
+			t.Fatalf("Error in key creation: %s", err)
```

Comment:
This and L130 should be `r.Fatalf` rather than `t` to use the retry correctly

Reply:
Doesn't look like the sample testutil supports  `Fatalf` (just `Errorf` & `Logf` functions)